pom.xml

```diff
@@ -164,6 +164,32 @@
   <build>
     <plugins>
 
+      <plugin>
+        <groupId>org.cyclonedx</groupId>
+        <artifactId>cyclonedx-maven-plugin</artifactId>
+        <version>2.7.0</version>
+        <configuration>
+          <projectType>library</projectType>
+          <schemaVersion>1.3</schemaVersion>
+          <includeBomSerialNumber>true</includeBomSerialNumber>
+          <includeCompileScope>true</includeCompileScope>
+          <includeProvidedScope>true</includeProvidedScope>
+          <includeRuntimeScope>true</includeRuntimeScope>
+          <includeSystemScope>true</includeSystemScope>
```

Comment:
I guess it doesn't really matter, since more information is better, but I'm trying to imagine why putting dependencies with these scopes in could matter for an SBOM.

Would a consumer care that we use lombok (provided scope), or transitive dependencies coming from lombok at compile time? Maybe? More of a question that a critique.